torchaudio/prototype/models/hdemucs.py

```diff
@@ -929,3 +929,55 @@ def _ispectro(z: torch.Tensor, hop_length: int = 0, length: int = 0, pad: int =
     _, length = x.shape
     other.append(length)
     return x.view(other)
+
+
+def hdemucs_low(sources: List[str], sample_rate: int) -> HDemucs:
+    r"""Builds low nfft (1024) version of HDemucs model. This version is suitable for lower sample rates, and bundles
+    parameters together to call valid nfft and depth values for a model structured for sample rates around 8 kHZ.
+
+    Args:
+        sources (List[str]): Sources to use for audio split
+        sample_rate (int): Serves as metadata, recommend lower sample rates.
+
+    Returns:
+        HDemucs:
+            HDemucs model.
+    """
+
+    return HDemucs(sources=sources, nfft=1024, depth=5, sample_rate=sample_rate)
+
+
+def hdemucs_medium(sources: List[str], sample_rate: int) -> HDemucs:
+    r"""Builds medium nfft (2048) version of HDemucs model. This version is suitable for medium sample rates,and bundles
+    parameters together to call valid nfft and depth values for a model structured for sample rates around 16-32 kHZ
+
+    Medium HDemucs has not been tested against the original Hybrid Demucs as there have been slight adjustments in
```

Comment:
has it not been tested or does it definitely produce different results? in the second case, might be better to explicitly state that. Also "slight adjustments" is vague

"Does not reach numerical parity with original HDemucs [link repo] because of  ..."

Reply:
oh nvm I remember these parameters fail to run on original demucs